include/wlr/types/wlr_xdg_output_manager.h

```diff
@@ -0,0 +1,32 @@
+#ifndef WLR_TYPES_WLR_XDG_OUTPUT_MANAGER_H
```

Comment:
Since the protocol is named `xdg-output`, I think it would make sense to name this `wlr_xdg_output.c`.

Reply:
I thought about that but I supposed that it makes more sense to name it after the primary exported interface